ecoscope/analysis/percentile.py

```diff
@@ -15,14 +13,14 @@ def _multipolygon(shapes, percentile):
 
 
 def get_percentile_area(
-    percentile_levels: typing.List,
+    percentile_levels: list,
```

Comment:
```suggestion
    percentile_levels: list[int],
```

Is this a list of ints? or can it be a list of ints or floats?

Just going off of the docstring below (which has a small capitalization typo).